### ISSUE TITLE
Parsers for a variety of TLD's

### ISSUE THREAD

**Issue by AlejoSanudo** [weppos/whois-parser#25]
No supported parser for:

.nyc
.london
.co.mz
.mm-registry.com
.work

So far, and the list grows ;)

**Comment by weppos** [comment]
Yes, I'm aware of that. Feel free to provide a patch.

**Comment by feens** [comment]
Should this be closed being as nothing has been done? It's technically still an issue, even if it's waiting on someone to provide a PR

**Comment by weppos** [comment]
Yes, it's an issue, but each missing parser is potentially an issue and I don't plan to have a ticket for each of them as it will only cause confusion.

It's virtually impossible to have 100% parser coverage, and having hundreds of "I want this parser" tickets living around forever is not gonna be useful at all.

If you want to open a PR with a parser attached, that is welcomed. Otherwise, tickets requesting a specific parser or reporting a specific parser as missing will be closed.

**Comment by feens** [comment]
> Yes, it's an issue, but each missing parser is potentially an issue and I don't plan to have a ticket for each of them as it will only cause confusion.

I can understand that you don't necessarily have the time to address all of them, but I would disagree with this statement.  As someone using the gem, it's a lot easier when I can see that a parser isn't supported (and that you know about it) if there's still a ticket.  And as a dev, it makes it easier to see what needs to be done if I'm going to contribute if there's issues listed.  Why not just have a tag in that case?

**Comment by weppos** [comment]
> As someone using the gem, it's a lot easier when I can see that a parser isn't supported 

There are no really "unsupported parsers". If the parser doesn't exist, it's just because it has not been created yet. And having a ticket around, is not going to help (it will just create confusion).

In fact, hostnames also change, and after one year we'll likely have dozens of invalid tickets. I went through this issue in the past.

Therefore I'm not going to keep parser tickets open. If you want to propose a new parser, it's just as easy as opening a new PR with code attached.
